Lib/ExecutionContext.cs

```diff
@@ -0,0 +1,151 @@
+// Copyright 2015 ThoughtWorks, Inc.
+
```

Comment:
The header is incorrect. This is from gauge-java. Please use the header from another gauge-csharp file.

Reply:
This has been corrected and pushed.  Thanks for catching this.  I will check the headers on my other pull request.